### PR TITLE
Migration of logging handlers to log4net (#69)

### PR DESCRIPTION
**Pull request by oleg-nenashev**
DONE: 
* Integrate log4net into WinSW, migrate wrapper.log engine

TODOs:
* Migrate EventLogs to log4net
* Delegate exceptions handling to log4net
* Allows setting up log levels from ServiceDescriptor
* (?) Migrate STDERR/STDOUT logs to log4net
* (?) Support XML configurations of log4net loggers

Signed-off-by: Oleg Nenashev <o.v.nenashev@gmail.com>

### CODE REVIEW

File: src/Core/ServiceWrapper/Main.cs [hunk 1]
```diff
@@ -542,8 +557,8 @@ public static void Run(string[] _args)
                     args = args.GetRange(2, args.Count - 2);
                 }
 
-                args[0] = args[0].ToLower();
-                if (args[0] == "install")
+                String command = args[0].ToLower();
+                if (command == "install")
```
Comment (oleg-nenashev) [inline src/Core/ServiceWrapper/Main.cs]: The solution occasionally adds the refactoring of the commands handling.
It needs to be reworked to have a proper merge with https://github.com/kohsuke/winsw

**Comment by oleg-nenashev** [comment]
Merging the basic logging to allow the usage of <code>ServiceWrapper</code> log in other dependencies.

**Comment by aminjam** [comment]
@oleg-nenashev We would like to enable sending all logs to Windows Event Viewer. Would Adding support for the above features `Migrate STDERR/STDOUT logs to log4net` and `Support XML configurations of log4net loggers` enable us to do that?

**Comment by oleg-nenashev** [comment]
@aminjam I doubt `Support XML configurations of log4net loggers` will be implemented in such way. #158 just addresses the minimal config. 

log4net already allows configuring everything in `exe.config` file, hence you will be able to get this feature once this issue is integrated (likely via the `logging=log4net` mode).

**Pull request merged**